FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilder/NewQueryBuilderFactoryInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder;
+
+use Shopware\Bundle\SearchBundle\Criteria;
+use Shopware\Bundle\StoreFrontBundle\Struct\ShopContextInterface;
+
+interface NewQueryBuilderFactoryInterface
```

Comment:
As I already noted in another comment, this interface is exactly the same as the `QueryBuilderFactoryInterface`. Additionally they actually should both implement the same interface, so we can swap out the old query builder with the new one (which is what we did).

Reply:
It is not the same. Since the interface comes from the Shopware namespace, it uses the QueryBuilderInterface, which has a QueryBuilder, and we are using NewQueryBuilder.

Reply:
Our old QueryBuilder also did not implement this interface and it worked without issues.